syntax = "proto3";

package hello;

service FraudService {
    rpc FraudLogic (FraudRequest) returns (FraudResponse);
}

message FraudRequest {
    string creditcardnr = 1;
}

message FraudResponse {
    string verdict = 1;
}
